### DISCUSSION TITLE
fire zone alerts not returned for point query

### DISCUSSION THREAD

**Discussion by JJKraw** [weather-gov/api#291] [Q&A]
I'm seeing this for a number of locations in Colorado where there are currently county, forecast zone, and fire zone alerts active for a number of locations.

A query of the form:

   https://api.weather.gov/alerts?active=1&point=40.4009,-105.0044

is not returning the fire zone alert (Red Flag Warning). In this example, COC069 and COZ038 (county and forecast zone) alerts are returned, but not the RF warning for COZ238.

A query for COZ238 directly or for the entire state of CO does return the fire zone alert.

**Comment by scadergit** [comment]
This would be considered an operational issue. Please open a support ticket so it can be escalated.
